azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -563,13 +574,14 @@ V1PodSpec createPodSpec(final int executionId, final VersionSet versionSet,
     // Get CPU and memory requested for a flow container
     final String flowContainerCPURequest = getFlowContainerCPURequest(flowParam);
     final String flowContainerMemoryRequest = getFlowContainerMemoryRequest(flowParam);
+    final String flowContainerDiskRequest = getFlowContainerDiskRequest(flowParam);
     logger.info("Creating pod for execution-id: " + executionId);
     final AzKubernetesV1SpecBuilder v1SpecBuilder =
         new AzKubernetesV1SpecBuilder(this.clusterEnv, Optional.empty())
             .addFlowContainer(this.flowContainerName,
                 azkabanBaseImageFullPath, ImagePullPolicy.IF_NOT_PRESENT, azkabanConfigVersion)
             .withResources(this.cpuLimit, flowContainerCPURequest, this.memoryLimit,
-                flowContainerMemoryRequest);
+                flowContainerMemoryRequest, flowContainerDiskRequest);
```

Comment:
curious about why there is "limit" and "request" values for CPU and memory but not for ephemeral storage.

Reply:
These are legacy params which will be removed later on. We do not want to have them separate that is why the multiplier is now 1. With storage we are not having any multiplier at all.

Reply:
got it, thanks!